packages/roosterjs-editor-dom/lib/utils/adaptFontColorToBackgroundColor.ts

```diff
@@ -0,0 +1,87 @@
+const enum ColorTones {
+    BRIGHT,
+    DARK,
+}
+const LiteralDarkColor = ['black', '--ms-color-black'];
+const LiteralBrightColor = ['white', '--ms-color-white'];
+const WHITE = '#ffffff';
+const BLACK = '#000000';
+
+//Using the HSL (hue, saturation and lightness) representation for RGB color values, if the value of the lightness is less than 20, the color is dark
+const DARK_COLORS_LIGHTNESS = 20;
+//If the value of the lightness is more than 80, the color is bright
+const BRIGHT_COLORS_LIGHTNESS = 80;
+
+/**
+ * @internal
+ * Change the font color to white or some other color, so the text can be visible with a darker background
+ * @param element The element that contains text.
+ * @param backgroundColor The backgroundColor of a element
+ */
+export default function adaptFontColorToBackgroundColor(
+    element: HTMLElement,
+    backgroundColor: string
+) {
+    if (element.firstElementChild?.hasAttribute('style')) {
+        return;
+    }
+    if (isADarkOrBrightColor(backgroundColor) === ColorTones.DARK) {
```

Comment:
No need to calculate twice.

const brightness = isADarkOrBrightColor(backgroundColor);
if (...)